web/pdf_viewer.js

```diff
@@ -387,6 +387,11 @@ class PDFViewer {
             if (--getPagesLeft === 0) {
               pagesCapability.resolve();
             }
+          }, (reason) => {
```

Comment:
The question in https://github.com/mozilla/pdf.js/pull/8746/files#diff-485f8990604b45b959a20e90265d8044R254 applies here as well.